app/templates/introduction.html

```diff
@@ -53,46 +53,21 @@ <h2 class="saturn col-10@l">{{legal_text}}</h2>
       </div>
     </div>
 
+    <div class="grid">
+        <div class="grid__col col-8@l">
   {% endblock business_details %}
 
-  {% block description %}
-    {% if content.block.description %}
-      <div class="u-mb-l qa-intro-description">
-        {{ content.block.description}}
-      </div>
-    {% endif %}
-  {% endblock description %}
-
-  {% block basis_for_completion %}
-    {% if content.block.basis_for_completion %}
-      <div data-qa="basis-for-completion">
-        <h2 class="saturn">Basis for completion</h2>
-        <ul>
-         {% for basis_item in content.block.basis_for_completion %}
-           <li>{{ basis_item }}</li>
-         {% endfor %}
-        </ul>
-      </div>
+  {% for intro in content.block.body %}
+    {% if intro.type == 'Basic' %}
+      {% include 'partials/intro-basic.html' %}
+    {% elif intro.type == 'Definition' %}
+      {% include 'partials/intro-highlight.html' %}
+    {% elif intro.type == 'Preview' %}
+      {% include 'partials/intro-preview.html' %}
```

Comment:
I think it would be worth having a `partials/introducton` folder and highlight should be `partials/introduction/definition.html`